app/uk/gov/hmrc/plasticpackagingtax/registration/views/phone_number_page.scala.html

```diff
@@ -59,5 +59,4 @@
             )
         @saveButtons()
     }
-  }
-)
\ No newline at end of file
+}
```

Comment:
Did you have a random `}` on your page?   Annoying that, isn't it?

Reply:
It was actually a random `)` ... see below line 63 :)
Very annoying